### PR TITLE
Enable armel for CI script

### PR DESCRIPTION
**Pull request by chunseoklee**
Signed-off-by: chunseoklee <chunseok.lee@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
This patch relates to https://github.com/dotnet/core-setup/issues/725

File: scripts/arm32_ci_script.sh [hunk 1]
```diff
@@ -166,14 +166,14 @@ function mount_emulator {
...
-    if [ "$__buildArch" == "arm-softfp" ]; then
+    if [ "$__buildArch" == "armel" ]; then
         #Export the needed environment variables
         (set +x; echo 'Exporting LINUX_ARM_* environment variable')
         source "$__ARMRootfsMountPath"/dotnet/setenv/setenv_incpath.sh "$__ARMRootfsMountPath"
 
         #Apply the changes needed to build for the emulator rootfs
-        (set +x; echo 'Applying cross build patch to suit Linux ARM emulator rootfs')
-        git am < "$__ARMRootfsMountPath"/dotnet/setenv/corefx_cross.patch
+        #(set +x; echo 'Applying cross build patch to suit Linux ARM emulator rootfs')
```
Comment (gkhanna79) [inline scripts/arm32_ci_script.sh]: Can you please remove the commented out lines if they are no longer required?

File: scripts/arm32_ci_script.sh [hunk 2]
```diff
@@ -212,7 +212,7 @@ do
         ;;
     --softfp)
```
Comment (gkhanna79) [inline scripts/arm32_ci_script.sh]: Why dont we change the `-softfp` to `armel` as well for consistency?

Comment (chunseoklee) [inline scripts/arm32_ci_script.sh]: I agree with you. 
It seems that `-softfp` option is passed by CI. To modify this name, we have to modify CI trigger command as well.

File: scripts/arm32_ci_script.sh (hunk 1 above)
Comment (chunseoklee) [inline scripts/arm32_ci_script.sh]: removed as you commented.

File: scripts/arm32_ci_script.sh (hunk 2 above)
Comment (chunseoklee) [inline scripts/arm32_ci_script.sh]: I found that we can modify `-softfp` in `corefx/netci.groovy` script.
I will update the last patch again soon.

**Comment by chunseoklee** [comment]
This patch conflicts to https://github.com/dotnet/corefx/pull/14844/ .
After https://github.com/dotnet/corefx/pull/14844/ is landed, I shall rebase on it.

**Comment by hqueue** [comment]
@chunseoklee I've updated  #14844 not to modify toolchain.cmake :)

File: scripts/arm32_ci_script.sh [hunk 3]
```diff
@@ -23,7 +23,7 @@ function usage {
     echo '    --buildConfig=<config>             : The value of config should be either Debug or Release'
     echo '                                         Any other value is not accepted'
     echo 'Optional Arguments:'
-    echo '    --softfp                           : Build as arm-softfp'
+    echo '    --armel                           : Build as armel'
```
Comment (mellinoe) [inline scripts/arm32_ci_script.sh]: nit: Spacing looks off by 1

**Comment by gkhanna79** [comment]
@chunseoklee Is this ready to merge?

**Comment by chunseoklee** [comment]
Except mellinoe's comment, it is ready to merge.

File: scripts/arm32_ci_script.sh (hunk 3 above)
Comment (chunseoklee) [inline scripts/arm32_ci_script.sh]: fixed as you commented

**Comment by chunseoklee** [comment]
@gkhanna79 ready to merged. Thank you !

**Pull request merged**